src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -164,6 +165,16 @@ struct ExportCoordinates {
           jac_diag.get(i));
     }
 
+    // Also output the computation domain metric
+    const auto& flat_logical_metric =
+        db::get<domain::Tags::FlatLogicalMetric<Dim>>(box);
```

Comment:
Is it really worth going through the DataBox infrastructure for this?  You already have the jacobian in this function, so it looks like about 50 lines of tag code could be removed by just calling `flat_logical_metric` here, like is done for `jacobian_diagnostic` above.

Reply:
I think we want to have the compute item to observe this quantity in other executables, so computing the quantity here directly only saves the couple of lines that add the compute item to the executable. Then I'd rather use the compute item here to be consistent with other execs that might use this.

Reply:
OK.